src/lib/dfp/dfp-env.ts

```diff
@@ -9,10 +9,8 @@ interface DfpEnv {
 	adSlotSelector: string;
 	lazyLoadEnabled: boolean;
 	lazyLoadObserve: boolean;
-	creativeIDs: string[];
-	advertIds: Record<string, number>;
 	advertsToLoad: Advert[];
-	adverts: Advert[];
+	adverts: Map<Advert['id'], Advert>;
```

Comment:
Should this be renamed to `advertsMap` or similar to reflect the type of object is represents?

Reply:
I personally think `adverts` is enough here, since the map is implied by the type. Happy to change it in the future if it proves confusing though.